### DISCUSSION TITLE
Add color_mode to light in exposes information?

### DISCUSSION THREAD

**Discussion by itavero** [Koenkk/zigbee2mqtt#22756] [General]
I noticed some lights have the `color_mode` property available to see what mode is currently used (e.g. temperature, xy or hs).

Unfortunately, I can't really find documentation on which devices provide this data and which do not.
It seems like an undocumented feature (or at least with hard-to-find docs).

Ideally I think this information should be added as a feature for type `light` in the exposes information for devices that support it.
That way I can also reliable decide in the homebridge-z2m codebase wether to take this property into account or not.

**Comment by Koenkk** [comment]
All lights supporting color temperature or color have the `color_mode`.
